src/features/dashboard/components/AppDashboardContainer/app-dashboard-container.scss

```diff
@@ -21,4 +21,10 @@
   &_main {
     width: 100%;
   }
+
+  @media screen and (min-width: 1024px) {
```

Comment:
@ahmed-deriv 

Let's create a common file and move all these mixins there for reusability.
For now this looks fine, but for later use let's go with this approach.